pipelines/rj_ssm__celular_seguro/flow.py

```
@@ -0,0 +1,33 @@
+    result = os.system(f"curl -v --connect-timeout 5 {url}")
+    print("Result:", result)
+
+
+    print("Testando diretamente pelo requests")
+    payload = {
+        "grant_type": "client_credentials",
+        "client_id": client_id,
+        "client_secret": client_secret
+    }
+
+    response = requests.post(url, json=payload, verify=False, timeout=30)
```

Ruff (0.15.20)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)